_posts/2016/03/2016-03-08-proposal-instructor-trainees.md

```diff
@@ -0,0 +1,83 @@
+layout: post
+authors: ["Belinda Weaver"]
+title: "A Proposal for Helping Instructor Trainees Finish"
+date: 2016-03-08
+time: "09:50:00"
+category: ["Instructor Training", "Mentoring"]
+Instructor training is one way we grow our community worldwide. Yet many people who
```

Comment:
Insert `---` above this line (after the 'category' line) to end the YAML block, or Jekyll won't know how to process the file.